### PR TITLE
Initial publish of fork

### PR DESCRIPTION
**Pull request by paulmelnikow**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Changelog
```
Comment (PyvesB) [inline CHANGELOG.md:1]: 👍

File: package.json [hunk 2]
```diff
@@ -4,14 +4,11 @@
...
-  "homepage": "http://espadrine.github.com/sc/",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/espadrine/sc.git"
-  },
+  "name": "@shields_io/camp",
+  "description": "Fork of Scoutcamp maintained for Shields",
+  "version": "18.0.0",
+  "homepage": "https://github.com/badges/sc",
+  "repository": "badges/sc",
```
Comment (PyvesB) [inline package.json:11]: The format of this parameter has changed. Any reason why? The previous type/url elements did seem consistent with the [official documentation](https://docs.npmjs.com/creating-a-package-json-file).

**Review by PyvesB — APPROVED** [review]
One small observation, not sure whether it matters much.

File: package.json (hunk 2 above)
Comment (paulmelnikow) [reply package.json:11]: Ah, good question. `repository` supports a shorthand for GitHub repos: https://docs.npmjs.com/files/package.json#repository.

It's in the spec and well-supported by tools.

File: CHANGELOG.md (hunk 1 above)
Comment (paulmelnikow) [reply CHANGELOG.md:1]: (Thanks @olivierlacan for hammering away at the importance of this one!)

File: package.json (hunk 2 above)
Comment (PyvesB) [reply package.json:11]: Yes, I didn't read that bit fully, thanks!

**Pull request merged**